src/lib/util/env.ts

```diff
@@ -2,4 +2,4 @@ export const rendererUrl: string =
 	(import.meta.env.MERMAID_RENDERER_URL as string) ?? 'https://mermaid.ink';
 export const krokiRendererUrl: string =
 	(import.meta.env.MERMAID_KROKI_RENDERER_URL as string) ?? 'https://kroki.io';
-export const debounceEnabled: boolean = import.meta.env.MERMAID_DISABLE_DEBOUNCE !== 'true';
+// export const debounceEnabled: boolean = import.meta.env.MERMAID_DISABLE_DEBOUNCE !== 'true';
```

Comment:
Should we delete the `MERMAID_DISABLE_DEBOUNCE` env in `package.json` and `test.yml` if it's not doing anything?

Maybe make it into a separate commit, so we can do a `git revert ...` if we ever do have to undo it.

Reply:
I pushed the branch with comments just to see if the CI passes. 
Have cleaned everything up.